src/test/java/api/cart/PostCartTests.java

```diff
@@ -15,4 +17,13 @@ public void createEmptyCart() {
         Cart postCartResponse = cartApiWrapper.postEmptycart(cart);
         Assert.assertNotNull(postCartResponse.getId(), "id is null");
     }
+
+    @Test
+    public void addItemsToCart() {
+        Cart cart = new Cart();
+        AddItemToCart itemToAdd = CartUtils.addItemToCart("30262", "5");
+        Cart postCartResponse = cartApiWrapper.postEmptycart(cart);
+        Cart postItemToCartResponse = cartApiWrapper.postCartItems(itemToAdd, postCartResponse.getId());
+        Assert.assertNotNull(postItemToCartResponse.getId(), "id is null");
```

Comment:
Please also add assert for status code, this check should be with highest priority.

Reply:
done